auth-web/src/components/auth/staff/continuation-application/AuthorizationContactInformation.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    v-if="!!review && !!filing"
+    id="authorization-contact-information"
+  >
+    <!-- Authorization Contact Information -->
+    <section class="section-container">
+      <v-row no-gutters>
+        <!-- First Column: Contact Information Label -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Contact Information</label>
+        </v-col>
+        <!-- Second Column: Email Address Label and Value -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Email Address</label>
+          <div>{{ email || '[Unknown]' }}</div>
+        </v-col>
+        <!-- Third Column: Phone Number Label and Value -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Phone Number</label>
+          <div>{{ phone || '[Unknown]' }}</div>
+        </v-col>
+      </v-row>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { ContinuationFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
+import { computed, defineComponent, reactive } from '@vue/composition-api'
+
+export default defineComponent({
+  name: 'AuthorizationContactInformation',
+
+  props: {
+    review: { type: Object as () => ContinuationReviewIF, required: true },
+    filing: { type: Object as () => ContinuationFilingIF, required: true }
+  },
+
+  setup (props) {
+    const state = reactive({
+      continuationIn: computed<any>(() => {
+        return props.filing?.continuationIn
+      }),
+
+      email: computed<string>(() => {
+        return state.continuationIn?.parties[0]?.officer?.email
+      }),
+
+      phone: computed<string>(() => {
+        return state.continuationIn?.parties[0]?.officer?.phone
+      })
```

Comment:
Could you please review this - is it correct here? It seems currently, phone info is not available.

Reply:
You should use the Contact Point object for the email and phone.

![image](https://github.com/user-attachments/assets/f5d0034d-3150-4885-a59f-3ae179977e6b)

Also, see if you can get rid of some conditional chaining characters (?). It's can be a double-edge sword: it can protect you, but it can also return a "valid" value in circumstances where you should do something else (like throw an error). It's probably safe here but my spine tingled when I saw this, haha.

Reply:
I see you don't display this component if you don't have a filing (line 3). Maybe you don't need to check that filing is non-null.

Reply:
Updated to use Contact Point object, and no need to check filing